Backpack/Toast/Classes/BPKToast.m

```diff
@@ -31,22 +31,31 @@ @interface BPKToast () <MBProgressHUDDelegate>
 @implementation BPKToast
 
 #pragma mark - Initializers
-+ (instancetype)showToastAddedTo:(UIView *)view animated:(BOOL)animated {
-    BPKToast *toast = [[BPKToast alloc] initWithView:view animated:animated];
++ (instancetype)showToastAddedTo:(UIView *)view
+                        animated:(BOOL)animated
+       accessibilityAnnouncement:(nonnull NSString *)accessibilityAnnouncement {
+    BPKToast *toast = [[BPKToast alloc] initWithView:view
+                                            animated:animated
+                           accessibilityAnnouncement:accessibilityAnnouncement];
     return toast;
 }
 
-- (instancetype)initWithView:(UIView *)view animated:(BOOL)animated {
+- (instancetype)initWithView:(UIView *)view
+                    animated:(BOOL)animated
+   accessibilityAnnouncement:(nonnull NSString *)accessibilityAnnouncement {
     if (self = [super initWithFrame:view.bounds]) {
         self.hud = [MBProgressHUD showHUDAddedTo:view animated:animated];
+        _accessibilityAnnouncement = [accessibilityAnnouncement copy];
         [self setupHUD];
+        [self postAccessibilityNotification];
```

Comment:
Should this announcement happen when the Toast is initialised? What if we initialise it but don't show it immediately?

Reply:
This is an internal init which is only called form `showToast:` but you are probably right that it'd be better to move this call

Reply:
Yeah just trying to future proof it